doc/conf.py

```diff
@@ -528,6 +528,7 @@
     'theme_logo_only': False,
     'current_version': version,
     'versions': (("latest", "/"),
+                 ("2.1.0", "/2.1.0/"),
```

Comment:
Since you are adding one, remove one from the bottom, so it's the same number of releases